docker/start.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+source scripts/common.sh
+source docker/docker_env.sh
+
+# start docker compose service list
+echo "start docker compose service: ${DOCKER_SERVICE_LIST[*]}"
+
+compose_up "${DOCKER_SERVICE_LIST[@]}"
+
+# check until l1 chain is ready
+until cast chain-id --rpc-url "$L2_EXECUTION_ENGINE_HTTP_ENDPOINT"; do
+    sleep 1
+done
+
+# show all the running containers
+echo
```

Comment:
is this empty `echo` necessary?

Reply:
This is optional in order to display an empty line.